CHANGES.md

```diff
@@ -69,6 +69,8 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 
 * Add Metrics for dns forwarding (DCOS-48336)
 
+* Add metrics for lashup (DCOS_OSS-4756)
```

Comment:
Should we also add? 
```
Add Metrics for dns workers (DCOS-49018)
````

Reply:
Let's remove `DCOS-48336` and add the epic/feature ticket when we merge all your work for dns metrics? It doesn't make much sense if all small tickets are mentioned in the change log.